testENSresolve.ts

```diff
@@ -0,0 +1,23 @@
+import { EdexaClient } from './src/index'
+
+async function resolveAddress(input: string) {
+    
+    let edexaclient = new EdexaClient();
+    let ens = await edexaclient.resolveAddr(input);
+    console.log("ens name of",input, " is:",ens)
+    
+    }
+async function resolveName(input : string) {
+    
+    let edexaclient = new EdexaClient();
+    let add = await edexaclient.resolveName(input);
+    console.log("address of",input, " is:",add)
+}
+
+
+
+resolveAddress("0x44aDbAcC4B0baD81dBc6D24b9512da3806c5aCCD")
```

Comment:
Add more test cases to check the functionality for example add negative test cases too